.env.example

```diff
@@ -10,11 +10,11 @@
 # PGPASSWORD=password
 
 # Action Mailer Configuration
-SMTP_PORT=1025
-SMTP_DOMAIN=localhost
-SMTP_AUTHENTICATION=plain
-SMTP_ADDRESS=localhost
-SMTP_ENABLE_TLS=false
+# SMTP_PORT=1025
+# SMTP_DOMAIN=localhost
+# SMTP_AUTHENTICATION=plain
+# SMTP_ADDRESS=localhost
+# SMTP_ENABLE_TLS=false
```

Comment:
Please break out system test step to it's own PR.

Reply:
Done! Plz review!